doap.xml

```diff
@@ -139,19 +139,66 @@
             <xmpp:status>complete</xmpp:status>
         </xmpp:SupportedXep>
     </implements>
+    <implements>
+        <xmpp:SupportedXep>
+            <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0422.html"/>
+            <xmpp:status>partial</xmpp:status>
+        </xmpp:SupportedXep>
+    </implements>
+    <implements>
+        <xmpp:SupportedXep>
+            <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0425.html"/>
+            <xmpp:status>complete</xmpp:status>
```

Comment:
I assume `complete`, but if you think we need to handle moderation (as admin/owner) from the app itself for saying it is `complete`, we should set `partial` instead

Reply:
You can set `partial` and add a note, `<xmpp:note>reading only</xmpp:note>`.